aexpect/client.py

```diff
@@ -277,6 +277,24 @@ def _close_reader_fds(self):
             except OSError:
                 pass
 
+    def _close_aexpect_helper(self):
+        """
+        Makes sure the subprocess holding the aexpect_helper is terminated
+        and that the file descriptors related to it are closed.
+        """
+        if self._aexpect_helper:
+            if self._aexpect_helper.stdout:
+                self._aexpect_helper.stdout.close()
+            try:
+                if self._aexpect_helper.stdin:
+                    self._aexpect_helper.stdin.close()
+            finally:
+                try:
+                    self._aexpect_helper.wait(10)
+                except subprocess.TimeoutExpired:
+                    self._aexpect_helper.kill()
+                    self._aexpect_helper.wait(10)
```

Comment:
And, at the end we could consider to release the `Popen` instances.
```python
self._aexpect_helper = None
```

Reply:
Yes, it looks way nicer like this, thank you